packages/headless/src/controllers/commerce/recommendations/headless-recommendations.test.ts

```diff
@@ -1,6 +1,7 @@
 import {contextReducer} from '../../../features/commerce/context/context-slice';
 import {fetchRecommendations} from '../../../features/commerce/recommendations/recommendations-actions';
 import {recommendationsReducer} from '../../../features/commerce/recommendations/recommendations-slice';
+import {configurationReducer} from '../../../features/configuration/configuration-slice';
```

Comment:
let's use the same import path
```suggestion
import {configurationReducer} from '../../../app/common-reducers';
```